templates/about/universal-operators.html

```diff
@@ -0,0 +1,86 @@
+<h3>Sidecar placement of operators</h3>
+<p>With the Juju OLM, it is also possible to create operators that have much more fine-grained control of their workload, by placing the operator code in a sidecar of the application workload pods.</p>
+<p>Sidecar operators can be thought of as running in ‘an adjacent chroot’ with the workload processes. They are guaranteed to be on the same machine as the workload container, and in fact they can share volumes directly since the same directory can be mounted into both the application container and the sidecar container in the pod. This enables richer interaction between the operator and the workload. For example, the operator might monitor status files, logs, or transaction logs in a very natural way.</p>
+<p>Inter-process communication between the operator and the workload becomes possible with sidecar operators, too, although the fact that the operator is in a separate container requires some adjusting. One can think of the operator as ‘running in a chroot’ on the same machine as the application; while this means they share a kernel, it also means that actual file or inter-process communication between the software in the workload and the software in the operator requires deliberate measures.</p>
+<h2>Operators for traditional machine-based applications</h2>
+<p>The lifecycle of traditional and legacy applications involves installation on one of the standard enterprise platforms &mdash; Ubuntu, CentOS, Windows and RHEL are all supported.</p>
+<p>Metadata in the operator package &mdash; the charm &mdash; declares the particular versions of the OS which are supported by that operator, for example, CentOS 6 and 7, or Ubuntu 18.04 LTS and 20.04 LTS, or Windows 10. When an operator is placed on a machine in a Juju OLM model, the Juju agent on that machine will fetch that operator from the controller and install it on that machine.</p>
+<p>Lifecycle events for this unit of the application are then delivered to the operator. Initial installation and configuration are driven by the first set of lifecycle events, followed by integration with related applications in the model. Upgrades of the application are driven through operator lifecycle events.</p>
+<h3>Operators for Linux applications</h3>
+<p>Linux machines have the Juju agent installed under /var/lib/juju/tools/ and the charms are installed under /var/lib/juju/agents/&lt;unit-app-number>/charm/. There can be multiple charms because multiple applications can have units co-located on the same machine, just as in traditional estate you can install as many apps on a machine as you want with your configuration management system. Subordinate application units will also follow their primary applications to every machine.</p>
+<p>And example directory tree for a machine in a Hadoop model looks like this:</p>
+
+<pre><code>
+  /var/lib/juju
```

Comment:
Can you move this to the previous line so that it doesn't have an empty line in the output?

![Screen Shot 2020-09-15 at 10 23 56 PM](https://user-images.githubusercontent.com/532033/93292178-2d00a100-f7a2-11ea-81b5-6915e0981694.png)